spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/ExportGenerator.java

```diff
@@ -134,7 +137,77 @@ public ExportGenerator withProduct(ProductDTO product, ContentDTO content) {
         return this;
     }
 
-    private File createExport(String consumerUuid, ExportCdn cdn) {
+    public ExportGenerator updateWithProduct() {
```

Comment:
This should not be done as an abstracted bit of logic at this level.

Really, the ExportGenerator should not exist at all, but if it's going to continue living on, it should be little more than boilerplate reduction and not inherit any of the old "does everything for everyone, everywhere" ideology that the original rspec code had.